torchci/pages/api/flaky-tests/disable.ts

```diff
@@ -0,0 +1,157 @@
+                octokit.rest.issues.createComment({
+                    owner,
+                    repo,
+                    issue_number: matchingIssue.number,
+                    body,
+                });
+            } else {
+                // Re-open the issue
+                octokit.rest.issues.update({
+                    owner,
+                    repo,
+                    issue_number: matchingIssue.number,
+                    state: "open",
+                });
+
+                const body = `Another case of flakiness has been found [here](${getLatestWorkflowURL(test.workflow_ids)}).
+                Reopening the issue to disable. Please verify that the platforms list includes all of
+                [${getPlatformsAffected(test.workflow_names).join(", ")}].`;
+                octokit.rest.issues.createComment({
+                    owner,
+                    repo,
+                    issue_number: matchingIssue.number,
+                    body,
+                });
+            }
+        } else {
+            await createIssueFromFlakyTest(test);
+        }
+    });
+}
+
+
+function getLatestWorkflowURL(workflow_ids: string[]): string {
+    return `https://github.com/pytorch/pytorch/actions/runs/${workflow_ids[workflow_ids.length - 1]}`;
+}
+
+
+function getIssueTitle(test_name: string, test_suite: string) {
+    let suite = test_suite;
+    // If the test class is not a subclass, it belongs to __main__
+    if (test_suite.indexOf(".") < 0) {
+        suite = "__main__." + suite;
+    }
+    return `DISABLED ${test_name} (${suite})`;
+}
+
+
+function getPlatformsAffected(workflow_names: string[]): string[] {
+    let platformsToSkip: string[] = [];
+    supportedPlatforms.forEach(function(platform: string) {
+        workflow_names.forEach(workflow_name => {
+            if (workflow_name.includes(platform) && !platformsToSkip.includes(platform)) {
+                platformsToSkip.push(platform);
+            }
+        });
+    });
+    return platformsToSkip;
+}
+
+
+function getIssueBodyForFlakyTest(test: FlakyTestData): string {
+    const examplesURL = `http://torch-ci.com/failure/${encodeURIComponent(`${test.name}, ${test.suite}`)}`;
+    const message = `Platforms: ${getPlatformsAffected(test.workflow_names).join(", ")}
+
+    This test was disabled because it is failing on trunk. See [recent examples](${examplesURL}) and the most recent
+    [workflow logs](${getLatestWorkflowURL(test.workflow_ids)}).
+
+    Over the past ${NUM_HOURS} hours, it has been determined flaky in ${test.workflow_ids.length} workflow(s) with
+    ${test.num_red} red and ${test.num_green} green.`;
+
+    return message;
+}
+
+
+async function getTestOwnerLabels(test_file: string) : Promise<string[]> {
+    const urlkey = "https://raw.githubusercontent.com/pytorch/pytorch/master/test/";
+
+    return await urllib.request(`${urlkey}${test_file}.py`).then(function (result): string[] {
```

Comment:
can probably use arrow function here as well. Also I don't believe you don't need the then, you can just get the result by
const result = await urllib.request();
...
return later;

Reply:
you can catch errors when awaiting by doing a try-catch statement